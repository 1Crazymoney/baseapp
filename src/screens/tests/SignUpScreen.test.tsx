import { shallow } from 'enzyme';
import * as React from 'react';
import { SignUpScreen } from '../';

describe('SignUpScreen test', () => {
    it('should render', () => {
        const wrapper = shallow(<SignUpScreen />);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toBeDefined();
    });
});
