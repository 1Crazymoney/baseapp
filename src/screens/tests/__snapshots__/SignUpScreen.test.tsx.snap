// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUpScreen test should render 1`] = `<SignUpScreen />`;
